src/plugins/openInNewTabAction/pluginSpec.js

```diff
@@ -66,6 +66,10 @@ describe("the plugin", () => {
             spyOn(window, 'open');
```

Comment:
For `resetApplicationState` to clean up this spy it has to know about it. If you use [`spyOnBuiltins(['open'])`](https://github.com/nasa/openmct/blob/43ac66233ef3bd17114cd635dc2ce6d01afd23c1/src/utils/testing.js#L47) it will know about it and clean it up when you call `resetApplicationState`.

Reply:
Hi @akhenry I've made the changes accordingly. Thanks!